packages/psss-server/.env.example

```diff
@@ -2,6 +2,7 @@ DB_NAME=
 DB_PASSWORD=
 DB_URL=
 DB_USER=
+ENTITY_API_URL=
```

Comment:
Hmm it's definitely called the `ENTITY_SERVER_API_URL` in the `lesmis-server` however I actually think I prefer your naming convention better... Perhaps if you or @tcaiger could change it there?

Reply:
I am following the existing convention in [MeditrakConnection](https://github.com/beyondessential/tupaia/blob/dev/packages/psss-server/src/connections/MeditrakConnection.ts#L10), [ReportConnection](https://github.com/beyondessential/tupaia/blob/dev/packages/psss-server/src/connections/ReportConnection.ts#L8)

Reply:
@rohan-bes I'm gonna merge this in the epic branch, and we can resolve the naming convention later